### PR TITLE
Password show hide

### PR DESCRIPTION
**Pull request by rko0211**
I have added the features of hide and show password in the log-in page  using Java-script code. You can see my work from the demo website : https://rko0211.github.io/Netflix-Sign-In-Page/.
if you have still any requirement in the page then kindly tell me i am ready to change the features. Thank you.
Fixes issue : #1

### CODE REVIEW

File: Password_show_hide.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+let password = document.getElementById("password");
+let lock = document.getElementById("lock");
```
Comment (Debaditya-Som) [inline Password_show_hide.js:2]: Nice one :)

**Review by Debaditya-Som — APPROVED** [review]

**Comment by Debaditya-Som** [comment]
Thanks for contributing to our project

**Comment by Debaditya-Som** [comment]
image

**Comment by rko0211** [comment]
> Thanks for contributing to our project

Thank you very much.

**Pull request merged**